src/main/kotlin/no/nav/familie/ba/sak/kjerne/eøs/endringsabonnement/TilpassValutakurserTilUtenlandskePeriodebeløpService.kt

```diff
@@ -76,3 +79,13 @@ internal fun tilpassValutakurserTilUtenlandskePeriodebeløp(
             }
         }.tilSkjemaer()
 }
+
+private fun korrigerValutakurserMedFremdtidigTom(gjeldendeValutakurser: Iterable<Valutakurs>): Iterable<Valutakurs> =
+    gjeldendeValutakurser
+        .mapNotNull {
+            when {
+                it.fom != null && it.fom.isAfter(YearMonth.now()) -> null
+                it.tom != null && it.tom.isSameOrAfter(YearMonth.now()) -> it.copy(tom = null)
+                else -> it
+            }
```

Comment:
Kan vurdere å ta i bruk [ClockProvider](https://github.com/navikt/familie-ba-sak/blob/main/src/main/kotlin/no/nav/familie/ba/sak/common/ClockProvider.kt) her om du ikke vil være coupled med systemklokken

```suggestion
            val nå = YearMonth.now(clockProvider.get()) // Mulig det finnes et bedre navn enn "nå" 
            when {
                it.fom != null && it.fom.isAfter(nå) -> null
                it.tom != null && it.tom.isSameOrAfter(nå) -> it.copy(tom = null)
                else -> it
            }
```